sonar-plugin-api/src/main/java/org/sonar/api/CoreProperties.java

```diff
@@ -191,8 +191,20 @@
   @Deprecated
   String CORE_ALLOW_USERS_TO_SIGNUP_PROPERTY = "sonar.allowUsersToSignUp";
 
+  /**
+   * @deprecated since 6.4. The default group is hardcoded to 'sonar-users'
+   * @see <a href="https://jira.sonarsource.com/browse/SONAR-9014">SONAR-9014/a>
+   */
+  @Deprecated
   String CORE_DEFAULT_GROUP = "sonar.defaultGroup";
```

Comment:
![INFO](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-info.png 'Severity: INFO') Do not forget to remove this deprecated code someday. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://next.sonarqube.com/sonarqube/coding_rules#rule_key=squid%3AS1133)